starwars/src/components/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './StarWars.css'
+
+function Card(props) {
+    const data = props.data
+    return (
+        <div style={{ color: "dark brown" }}>
+        <div className="card">
+            <h1>{data.name}</h1>
+            <div>
```

Comment:
This div isn't necessary either, unless you want to add a class to differentiate some styling between the heading and the character stats.